instructors.md

```diff
@@ -3,28 +3,126 @@ layout: page
 title: Version Control with Git
 subtitle: Instructor's Guide
 ---
-## Legend
+Version control might be the most important topic we teach,
+but Git is definitely the most complicated tool.
+However,
+GitHub presently dominates the open software forge landscape,
+so we have to help novices learn just enough Git
+to feel they can and should learn more on their own.
 
-Wolfman and Dracula will work together in one project.
+This is why we don't teach branching:
```

Comment:
> How long is the workshop?

The Git section normally take one morning or one afternoon.

> I've been (I think) quite successful in teaching branching and PRs in a 4h session...

Depending of how much challenges do you leave the students do and how many helpers do you have will be possible to teach branching but we mostly agree that this is a intermediate feature so we prefer not teaching it for novices.